play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcReportTechnicalIssueHelperSpec.scala

```diff
@@ -77,7 +77,7 @@ class hmrcReportTechnicalIssueHelperSpec
       links should have size 1
       links.first.attr(
         "href"
-      )   shouldBe "https://www.tax.service.gov.uk/contact/report-technical-problem?newTab=true&service=online-payments&referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Ffoo"
+      )   shouldBe "https://www.tax.service.gov.uk/contact/report-technical-problem?service=online-payments&referrerUrl=https%3A%2F%2Fwww.tax.service.gov.uk%2Ffoo"
```

Comment:
jsoup is decoding it for us too! But I think it's ok in this case because we've got the fixture tests and the test in hmrc-frontend to ensure it's encoded correctly